pkg/cli/walletcommands.go

```diff
@@ -142,6 +141,12 @@ func transferCMD(args []string, publisher wire.EventBroker, rpcBus *wire.RPCBus)
 		return
 	}
 
+	// Sync wallet
+	if err := syncWallet(); err != nil {
+		fmt.Fprintf(os.Stdout, "%v", err)
```

Comment:
why the error is printed and not returned? The function clearly returns `(*Wallet, error)`